### ISSUE TITLE
Is there any way to directly train and reproduce the results with ucf101 data set. 

### ISSUE THREAD

**Issue by Aldebaranus** [Phoenix1327/tea-action-recognition#8]
Could you give some suggestions on setting hyperparameters. I only have a 2080ti GPU。

**Comment by Phoenix1327** [comment]
Please refer to [issue4](https://github.com/Phoenix1327/tea-action-recognition/issues/4). Basically, you can utilize a small batch size and correspondingly reduce the learning rate.

**Comment by zkyunqi** [comment]
Hello, do you reproduce the result with ucf101 data set?
